nomic/data_operations.py

```diff
@@ -846,3 +846,125 @@ def remove(self, ids: List[str], tags: List[str], delete_all: bool = False) -> b
 
     def __repr__(self) -> str:
         return str(self.df)
+
+
+class AtlasMapData:
+    """
+    Atlas Map Data (Metadata) State.
+    This is how you can access text and other associated metadata columns
+    you uploaded with your project.
+
+    === "Accessing Data Example"
+        ``` py
+        from nomic import AtlasProject
+
+        project = AtlasProject(name='My Project')
+        map = project.maps[0]
+        print(map.data)
+        ```
+    === "Output"
+        ```
+                        id_                        text                   title
+        0     000262a5-2811   The Hurricane occurred...        Hurricane Jeanne
+        1     000c453d-ee97    In the football games...    Athens 2004 Olympics
+        ...
+        9999  fffcc65c-38dc  In 2012, the candidates...  Presidential elections
+        ```
+    """
+
+    def __init__(self, projection: "AtlasProjection"):
+        self.projection = projection
+        self.project = projection.project
+        self.id_field = self.projection.project.id_field
+        try:
+            # Run fetch_tiles first to guarantee existence of quad feather files
+            self._tb: pa.Table = self.projection._fetch_tiles()
```

Comment:
I'm having a little trouble following this, but IMO we should do here 
```
self._tb = None # Reserve this name for the actual table of all the data returned here.
try:
    self._basic_data = self.projection._fetch_tiles()
# Maybe drop the topics because they're different
```
b/c 1. that returns an arrow table.